pkl-config-java/gradle.lockfile

```diff
@@ -1,19 +1,18 @@
 # This is a Gradle generated file for dependency locking.
 # Manual edits can break the build and are not advised.
 # This file is expected to be part of source control.
-com.github.ajalt.clikt:clikt-jvm:3.5.1=pklCodegenJava
-com.github.ajalt.clikt:clikt:3.5.1=pklCodegenJava
+com.github.ajalt.clikt:clikt-jvm:3.5.4=pklCodegenJava
+com.github.ajalt.clikt:clikt:3.5.4=pklCodegenJava
 com.squareup:javapoet:1.13.0=pklCodegenJava
-com.tunnelvisionlabs:antlr4-runtime:4.9.0=default,pklCodegenJava,runtimeClasspath,testRuntimeClasspath
-io.leangen.geantyref:geantyref:1.3.14=default
-io.leangen.geantyref:geantyref:1.3.15=compileClasspath,implementationDependenciesMetadata,runtimeClasspath,testCompileClasspath,testImplementationDependenciesMetadata,testRuntimeClasspath
+com.tunnelvisionlabs:antlr4-runtime:4.9.0=pklCodegenJava,runtimeClasspath,testRuntimeClasspath
+io.leangen.geantyref:geantyref:1.3.16=compileClasspath,implementationDependenciesMetadata,runtimeClasspath,testCompileClasspath,testImplementationDependenciesMetadata,testRuntimeClasspath
```

Comment:
Looks like geantyref doesn't have a changelog or release notes right now; filed https://github.com/leangen/geantyref/issues/31

Looking at the git diffs, though, should be fine to bump to 1.3.16.